.travis.yml

```diff
@@ -32,7 +32,15 @@ cache:
 branches:
   only:
     - develop
-
-script:
-  - pre-commit run -a && (cd cgotorch; make) && go install ./...
-  - go test -v ./...
+jobs:
+  include:
+  - env: ARCH=MacOS
+    script:
+    - set -e
+    - bash ./scripts/build_and_test.sh
+  - env: ARCH=Linux
```

Comment:
I created the Dockerfile only because I don't have a Linux computer by my hand so I use the Docker container as a Linux VM.

As we know how to build and run on Linux now, we can use Travis Linux VM/container to replace the macOS VM, as the it's faster to start a Linux VM/container than the macOS VM.